bcgov/biohubbc
# SIMSBIOHUB-177: Survey Study Area enhancements

## Links to Jira Tickets

[SIMSBIOHUB-177](https://apps.nrs.gov.bc.ca/int/jira/browse/SIMSBIOHUB-177)

## Description of Changes
- Survey Study area can now import files with multiple features
- Drawn features can be created and saved to a survey
- New card for displaying and editing map features
- Region selector updated

## Testing Notes

- Create/ Edit a survey
- Use the import feature to upload a list of features
- Use the draw controls to add new geometry to the map
- Modify the new geometry and save it (using the draw controls)
- Update any of the metadata (below the map)
- Save the survey
- There is no front end component that shows these so you'll need to edit the survey you just modified to verify the changes
